comp/dogstatsd/server/parse.go

```diff
@@ -238,8 +246,25 @@ func (p *parser) parseFloat64List(rawFloats []byte) ([]float64, error) {
 }
 
 // extractContainerID parses the value of the container ID field.
+// if the field starts with 'in-' it means that we want to resolve
+// the origin detection using Inodes.
```

Comment:
```suggestion
// If the field is prefixed by `in-`, it corresponds to the cgroup controller's inode of the source
// and is used for ContainerID resolution.
```
I think we might prefer to avoid using `we` in the comments and be more a bit more precise